tests/model_serving/model_server/ovms/model_mesh/conftest.py

```diff
@@ -28,23 +28,37 @@
 
 
 @pytest.fixture(scope="class")
-def http_s3_openvino_model_mesh_serving_runtime(
+def http_s3_ovms_model_mesh_serving_runtime(
+    request: FixtureRequest,
     admin_client: DynamicClient,
     ns_with_modelmesh_enabled: Namespace,
 ) -> ServingRuntime:
-    with ServingRuntimeFromTemplate(
-        client=admin_client,
-        name=f"{Protocols.HTTP}-{ModelInferenceRuntime.OPENVINO_RUNTIME}",
-        namespace=ns_with_modelmesh_enabled.name,
-        template_name=RuntimeTemplates.OVMS_MODEL_MESH,
-        multi_model=True,
-        resources={
+    rt_kwargs = {
```

Comment:
nitpick, `rt` isn't immediately clear as an abbreviation, a more meaningful name would be better

Reply:
renamed in following pr